package/network/config/ltq-adsl-app/Makefile

```diff
@@ -36,6 +36,7 @@ define Package/ltq-adsl-app
   TITLE:=Lantiq DSL userland tool
   URL:=http://www.lantiq.com/
   DEPENDS:=@(TARGET_lantiq_xway||TARGET_lantiq_xway_legacy||TARGET_lantiq_ase) +libpthread +ltq-dsl-base +libubox +libubus
+  PROVIDES:=ltq-dsl-app
```

Comment:
just for reference the tool export the same stuff or we can encounter some compatibility problem?

Reply:
There may be `ltq-vdsl-vr11-app` in the future, and with that all three are from the same codebase, just different versions supporting different hw generations (think branches).

The primary use is for consumers for the ubus stats those three provide, at this point that's `luci-mod-status`, `prometheus-node-exporter-lua` and `collectd`. Luci detects it at runtime, but one contains `DEPENDS:=@(PACKAGE_ltq-adsl-app||PACKAGE_ltq-vdsl-app)` and the other `@TARGET_lantiq_xrx200`. With ipq40xx/ltq-vdsl-vr11-app on top that'll get a bit ugly, so this is for them switching to just `ltq-dsl-app`.

I was using `prometheus-node-exporter-lua` on lantiq/vr9 and now I'm using it on ipq40xx/vr11, and as far as I can tell both provide the very same stats for my dsl line, so I don't expect any compatibility problems.